### PR TITLE
Fixes 1759: add 9.2 repos

### PR DESCRIPTION
**Pull request by jlsherrill**
## Summary
cherry pick from main

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMS-1759

File: pkg/external_repos/external_repos.json [hunk 1]
```diff
@@ -53,6 +53,12 @@
     {
         "base_url": "https://cdn.redhat.com/content/dist/rhel9/9.1/x86_64/baseos/os"
     },
+    {
+        "base_url": "https://cdn.redhat.com/content/dist/rhel9/9.2/x86_64/appstream/os"
```
Comment (mshriver) [inline pkg/external_repos/external_repos.json:57]: The image-builder distribution definition includes `aarch64`, looks like in general we're only setting up introspection of x86_64?

Comment (jlsherrill) [reply pkg/external_repos/external_repos.json:57]: correct, they only support x86_64 currently in the WebUI today

**Review by mshriver — APPROVED** [review]

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMS-1760

**Pull request merged**